tool/actions-plan-preview/main.go

```diff
@@ -139,6 +139,9 @@ func main() {
 			doComment(failureBadgeURL + "\nUnable to run plan-preview for a closed pull request.")
 			return
 		}
+		body := makeCommentBody(event, result)
+		doComment(failureBadgeURL + "\n" + body)
+		log.Fatal("plan-preview result has error")
```

Comment:
This case is an expected error. Therefore, it is inappropriate to set the log level to fatal.
If you just want to call exit(1), I think you can simply call exit(1).

Reply:
@kurochan 
You mean fatal log level must be used when unexpected error, right?
I agree with your point 👍
I'll fix later, thanks.

Reply:
@kurochan 
Applied!!
https://github.com/pipe-cd/pipecd/pull/4662/commits/23a79cdea1772ebe9512a799ecd87c872d4604d3